src/api/component.js

```diff
@@ -11,6 +11,9 @@ import debounce from '../util/debounce';
 import getAllKeys from '../util/get-all-keys';
 import getOwnPropertyDescriptors from '../util/get-own-property-descriptors';
 
+const setElementAsDefined = elem => elem.setAttribute('defined', '');
+const hasReflect = 'Reflect' in window;
```

Comment:
Should we move this to [support](https://github.com/skatejs/skatejs/blob/master/src/util/support.js) then?